<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="INW1PF1PY3M7" name="Test" tests="3" failures="2" timestamp="22 Jan 2020 18:12:01 GMT" time="559.505" errors="0">
  <testcase name="LogIntest" time="16.083" classname="com.hellofresh.challenge.WebTest"/>
  <testcase name="CheckOuttest" time="54.531" classname="com.hellofresh.challenge.WebTest">
    <failure type="java.lang.AssertionError" message="expected [MY ACCOUNT] but found [WOMEN 
There are 7 products.]">
      <![CDATA[java.lang.AssertionError: expected [MY ACCOUNT] but found [WOMEN 
There are 7 products.]
	at com.hellofresh.challenge.CheckOutPage.checkoutTest(CheckOutPage.java:52)
	at com.hellofresh.challenge.WebTest.CheckOuttest(WebTest.java:68)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- CheckOuttest -->
  <testcase name="signInTest" time="543.434" classname="com.hellofresh.challenge.WebTest">
    <failure type="java.lang.AssertionError" message="expected [MY ACCOUNT] but found [WOMEN 
There are 7 products.]">
      <![CDATA[java.lang.AssertionError: expected [MY ACCOUNT] but found [WOMEN 
There are 7 products.]
	at com.hellofresh.challenge.SignUpPage.SignInSuccessAsserts(SignUpPage.java:93)
	at com.hellofresh.challenge.WebTest.signInTest(WebTest.java:45)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- signInTest -->
</testsuite> <!-- Test -->
